inc/Modules/Search/Algolia.php

```diff
@@ -19,23 +18,11 @@
  * Class - Algolia
  */
 final class Algolia {
```

Comment:
Now it's a pure service wrapper, no more singleton or caching. Direct usage through the app is now via `Index->get_index()` instead of this class.